apps/heft/src/cli/actions/StartAction.ts

```diff
@@ -31,6 +32,13 @@ export class StartAction extends HeftActionBase {
       parameterLongName: '--clean',
       description: 'If specified, clean the package before starting the development server.'
     });
+
+    // TODO: Expose an API for custom CLI parameters similar to HeftSession.registerAction()
```

Comment:
@halfnibble - have you had a chance to take a look at this?

Reply:
I didn't notice the email about this since it went to my personal email. I'm looking now.

Reply:
I'm proposing that we should merge this PR with the `--storybook` hack, and then @halfnibble's PR can replace it with `registerParameters()` to validate that the API design works.